addon-test-support/@ember/test-helpers/dom/blur.js

```diff
@@ -0,0 +1,37 @@
+import getElement from './-get-element';
+import fireEvent from './fire-event';
+import settled from '../settled';
+import isFocusable from './-is-focusable';
+
+/*
```

Comment:
doc comments should use `/**` instead of `/*`. this seems to be the case for the majority of the doc comments in this PR.

Reply:
Good catch. Will update.